src/components/Testimonials/TestimonialItems.js

```diff
@@ -7,19 +7,20 @@ import s from './Testimonials.css'
 
 type Props = {
   item: [],
-  key: '',
+  key: [],
```

Comment:
The key is a number, not an array. But anyway I don't think it's needed (see comment below about using title as key)